packager/certcache/certcache_test.go

```diff
@@ -111,8 +112,10 @@ func (this *CertCacheSuite) TearDownSuite() {
 
 func (this *CertCacheSuite) SetupTest() {
 	this.fakeClock = pkgt.NewFakeClock()
+        this.fakeClock.SecondsSince0 = pkgt.B3Certs[0].NotBefore.Sub(time.Unix(0, 0))
+        now := this.fakeClock.Now()
 	var err error
-	this.fakeOCSP, err = FakeOCSPResponse(this.fakeClock.Now())
+	this.fakeOCSP, err = FakeOCSPResponse(now, now.Add(1*time.Minute))
```

Comment:
Could you add a comment here as to why a minute was added?

Reply:
Good idea. Decided to extract it into the FakeOCSPResponse function so there was only one place to document it.